app/pages/nonsipp/otherassetsdisposal/AnyPartAssetStillHeldPage.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.otherassetsdisposal
+
+import utils.RefinedUtils._
+import models.SchemeId.Srn
+import play.api.libs.json.JsPath
+import config.Refined.{Max50, Max5000}
+import pages.QuestionPage
+
+case class AnyPartAssetStillHeldPage(srn: Srn, assetIndex: Max5000, disposalIndex: Max50)
```

Comment:
@narendrapaduchuri I'm not sure if there is an AC for this, but do we have to do the following?
If any asset disposal of an asset has a value of 'false' for AnyPartAssetStillHeldPage, the asset should not be visible on the 'Start reporting an assey disposal' page .